app/enterprise/1.3-x/kong-manager/administration/admins/invite.md

```diff
@@ -35,74 +20,29 @@ the appropriate **Role** and click **Invite User** to send the invitation.
     ⚠️ **IMPORTANT**: The **Super Admin** can see all available roles across 
     **Workspaces** on the **Roles** tab of the **Organization** page. 
 
-    ![Role List](https://konghq.com/wp-content/uploads/2018/12/org2.png)
-
-3. On the **Organization** page, the new invitee will appear on the list with the 
-**Invited** status. Once they accept the invitation, the user will be listed in 
-the main **Users** list. 
-
-    ![User List](https://konghq.com/wp-content/uploads/2018/12/org3-1.png)
-
-4. The newly invited **Admin** will have the ability to set a password. If the 
-**Admin** ever forgets the password, it is possible for them to reset it through a 
-recovery email.
-
-<video width="100%" autoplay loop controls>
-  <source src="https://konghq.com/wp-content/uploads/2019/02/org-super-admin-ent-34.mov" type="video/mp4">
-  Your browser does not support the video tag.
-</video>
-
-## How to Invite a New Admin in a Workspace
-
-![Create an Admin in Kong Manager](https://konghq.com/wp-content/uploads/2018/07/admins2.png)
-
-1. On the "Admins" page, to invite a new admin, click the "Create New Admin" 
-button.
-
-2.  When a new Admin receives an invitation, they will only be able to log in 
-    with that email address. Assign the appropriate Role and click "Invite Admin" 
-    to send the invitation.
-
-    ![Create New Admin](https://konghq.com/wp-content/uploads/2018/11/km-name-admin.png)
-
-3. On the "Admins" page, the new invitee will appear on the list with the 
-    "invited" status. Once they accept the invitation, their status will 
-    change to "accepted".
 
-    ![Invited Admins](https://konghq.com/wp-content/uploads/2018/11/km-invited-admins.png)
+4. On the **Teams** page, the new invitee will appear on the **Admins** list with the under **Invited**. Once they accept the invitation, the user will be listed in the main **Admins** list. 
 
-4. The newly invited Admin will have the ability to set a password. If the 
-    **Admin** forgets the password, it is possible to reset it through a recovery 
-    email.
+    ![Invited Admins](https://doc-assets.konghq.com/1.3/manager/teams/teams-invited-admin.png)
 
-⚠️ **IMPORTANT**: By default, the registration link will expire after 259,200 
```

Comment:
All of the warnings on 78-87 got added due to past issues. 

Maybe we can move them to a "known issues" subsection at the bottom, just for coverage.

Reply:
Got it, I will add this back in